model_analyzer/config/generate/base_model_config_generator.py

```diff
@@ -212,6 +199,38 @@ def make_model_config(
 
         return model_config
 
+    @staticmethod
+    def make_ensemble_model_config(
+            model: ModelProfileSpec, ensemble_submodels: List[ModelConfig],
+            model_variant_name_manager: ModelVariantNameManager) -> ModelConfig:
+        """
+        Loads the base model config from the model repository, and then mutates
+        the names to match the ensemble submodels
+        """
+        model_name = model.model_name()
+        model_config_dict = model.get_default_config()
+        ensemble_config_dicts = [
+            model.to_dict() for model in ensemble_submodels
```

Comment:
seems bad to reuse 'model' variable name to be the same as one passed in

Reply:
Changed to submodel